src/apps/chifra/cmd/serve/options.go

```diff
@@ -13,12 +13,15 @@ package utils
  * Public License along with this program. If not, see http://www.gnu.org/licenses/.
  *-------------------------------------------------------------------------------------------*/
 
-var Off string = "\033[0m"
-var Red string = "\033[31m"
-var Green string = "\033[32m"
-var Yellow string = "\033[33m"
-var Blue string = "\033[34m"
-var Purple string = "\033[35m"
-var Cyan string = "\033[36m"
-var White string = "\033[37m"
-var Black string = "\033[30m"
+// OptionsType makes command line options available to the package
+type OptionsType struct {
+	Scrape  bool
```

Comment:
I think (but I'm not sure) that both `Scrape` and `Monitor` can be removed from this structure. These are hold-overs from the old scraper code which was removed from the old server code a while ago.